mesosutil/mesosprotoutil.go

```diff
@@ -36,6 +36,15 @@ func FilterResources(resources []*mesos.Resource, filter func(*mesos.Resource) b
 	return result
 }
 
+func AddResourceRevocable(resource *mesos.Resource, principal string, role string) *mesos.Resource {
+        resource.Revocable = &mesos.Resource_RevocableInfo{}
+        return resource
+}
+
+func NewScalarResourceWithRevocable(name string, value float64, principal string, role string) *mesos.Resource {
```

Comment:
I'm not crazy about propagating this anti-pattern of func-name-for-every-resource-type-and-feature. This kind of API nonsense will disappear in mesos:next.